extensions/amp-consent/0.1/test/test-consent-state-manager.js

```diff
@@ -414,6 +434,12 @@ describes.realWin('ConsentStateManager', {amp: 1}, (env) => {
         instance.update(CONSENT_ITEM_STATE.ACCEPTED, 'test', {});
         yield macroTask();
         expect(storageSetSpy).to.be.calledTwice;
+        instance.update(CONSENT_ITEM_STATE.ACCEPTED, 'test');
```

Comment:
metadata `{}` and `undefined` is different?

Reply:
Yes, consentMetadata from server response could be `undefined` and `{}` resulting in `undefined` or empty `ConsentMetadataDef` respectfully. Should we treat these as the same (aka`undefined` response generates an empty `ConsentMetadataDef`)?

Reply:
Depends on if `undefined` and `{}` will be treated differently by vendors, and in localStorage. 
My understanding is that the two value (and even `null`) means the same thing.

Reply:
Ok do you think it's ok if I leave it as is, incase vendor does need that distinction?